executor/aggregate_xapi_test.go

```diff
@@ -13,483 +13,484 @@
 
 package executor
```

Comment:
@hanfei1991 Please remove this file and add tests for NewXAggregateExec in this PR or in the  next PR.